.env.example

```
@@ -10,8 +10,11 @@ BASE_OS=alpine
 BASE_VERSION=3.23
 BUN_VERSION=1.3.3
 DB_VERSION=17.7
+
 FORWARD_APP_PORT=3010
 FORWARD_DB_PORT=5432
+FORWARD_MAIL_PORT=1025
+FORWARD_MAILIT_PORT=8025
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FORWARD_MAILIT_PORT key should go before the FORWARD_MAIL_PORT key

(UnorderedKey)